### ISSUE TITLE
md-subheader don't allow clicks or hover events for md-button inside

### ISSUE THREAD

**Issue by mattiarigo-humus** [angular/material#3932]
Hello,
I'm using 0.9.8 because 0.10.0 still has some little bugs.
When an md-button is inside an md-subheader there is no HOVER event for css rules neither click event.
I've tried this css solution

```
.md-subheader .md-subheader-content > *{
    position: relative;
    z-index: 10;
}
```
this rule works but it give me this on the page when scrolling

![schermata 2015-07-28 alle 17 49 52](https://cloud.githubusercontent.com/assets/5363395/8936175/1a325a14-3551-11e5-9dc0-ab89623be429.png)

I've no hover event or click event for any of element inside md-subheader.
How can I fix it?

Thanks in advance,
\m

**Comment by marcysutton** [comment]
A Codepen or Plunker would help diagnose this issue.

**Comment by pratyushmittal** [comment]
@marcysutton Here is a Codepen demo: http://codepen.io/anon/pen/xGQPyw

**Comment by topherfangio** [comment]
@pratyushmittal I need to do some more testing to make sure it doesn't break anything else, but it appears you can add the following CSS to fix the issue:

```css

.md-subheader .md-subheader-content {
  display: block;
}
```

Can you give it a try? If all looks good, we'll get this added/fixed in 0.12.0.

**Comment by pratyushmittal** [comment]
@topherfangio That seems to be working fine. Thanks.
